src/openassetio-python/cmodule/src/_openassetio.hpp

```diff
@@ -20,10 +20,33 @@ OPENASSETIO_FWD_DECLARE(log, LoggerInterface)
 OPENASSETIO_FWD_DECLARE(hostApi, HostInterface)
 OPENASSETIO_FWD_DECLARE(hostApi, ManagerImplementationFactoryInterface)
 
+/// @note Update errorsTest.cpp if adding more override macros below.
+
+/**
+ * Decorate PYBIND11_OVERRIDE_NAME with exception type translation.
+ */
+#define OPENASSETIO_PYBIND11_OVERRIDE_NAME(ret_type, cname, name, fn, ...)                      \
```

Comment:
[Consider] I might think about defining these in their own place within the cmodule dir, `exception_translation_macros.hpp` or something. They get in the way of the "point" of this file imo.

Reply:
Good idea, they are using up a lot of space now. The `_openassetio.hpp` has become a bit of a dumping ground.

Reply:
Good call. Put in their own header in a8e8db56 - much nicer and YAGNI.